modules/ROOT/pages/core-operations.adoc

```diff
@@ -42,22 +42,22 @@ Because JSON is a structured format, it can be subsequently searched and queried
 [#crud-overview]
 == Primitive Key-Value Operations
 
-[source,python]
+[source,java]
 ----
-upsert(docid, document)
-insert(docid, document)
-replace(docid, document)
-get(docid)
-remove(docid)
+bucket.upsert(document);
+bucket.insert(document);
+bucket.replace(document);
+bucket.get(docid);
+bucket.remove(docid);
```

Comment:
As elsewhere, these examples are too incomplete, they need to include creating the JsonDocument.

Consider what would be helpful to you as a developer.  E.g. the get example could show `JsonDocument result = bucket.get(docid);` instead.  Feel free to go a little above and beyond the Python snippets, which are a bit bare-bones.

Reply:
Details explanations have already been given in relevant section on page https://docs.couchbase.com/java-sdk/2.7/document-operations.html. Will mention the links for reference.